ko/sql/schema/index_stmt.rst

```diff
@@ -224,6 +237,71 @@ CREATE INDEX
 |                                                                   |                                                                                   |
 +-------------------------------------------------------------------+-----------------------------------------------------------------------------------+
 
+.. _deduplicate_overview:
+
+DEDUPLICATE 
+-----------
+
+**DEDUPLICATE** 옵션을 사용하면 특정 키값에 치우쳐진 비대칭 데이터(skewed data)에 의해 인덱스 데이터 수정 시 발생할 수 있는 성능 저하를 개선할 수 있다. 이 옵션 값으로 특정 키 값에 대한 OID 오버플로우 페이지의 연결 리스트가 너무 길게 만들어지지 않게 완화하도록 조정할 수 있으며 그에 따라 삽입/삭제/갱신 및 vacuum 성능을 개선한다. 하지만 시스템에 의해 숨겨진 인덱스 칼럼이 추가되었으므로 인덱스의 단말 노드의 개수와 트리의 높이가 증가 할 수 있어 인덱스의 크기가 커지며 조회 성능에 영향을 줄 수 있으므로 주의헤야 한다. 특히, 인덱스 데이터가 키 값에 대하여 균일하게 분포하는 경우에는 성능 개선 효과 없이 인덱스의 크기만 커질 수 있으므로 주의해야 한다.
```

Comment:
We need to have some explanation on the term "OID 오버플로우 페이지의 연결 리스트". 
Or, we have to give the document without the term.

Reply:
I agree with Hyunik's opinion too. In the previous PR, the term, OID overflow pages, was already introduced in the part of vacuumdb utility. So It would be better to explain the OID overflow pages structure somewhere and it needs to be referenced.

Reply:
Added explanation to overflow page.